desktop/extensions-sdk/guides/use-docker-socket-from-backend.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Use the Docker socket from the extension backend
+description: Docker extension metadata
+keywords: Docker, extensions, sdk, metadata
+---
+
+Docker extensions can invoke Docker commands directly from the frontend with the SDK. In some cases, it is useful to also
+interact with the Docker engine from the backend. Extension backend containers can mount the Docker socket and use it to
+interact with the Docker engine from the extension backend logic. (Learn more about the [Docker engine socket](/engine/reference/commandline/dockerd/#examples))
+
+However, when mounting the Docker socket from an extension container that lives in the Desktop virtual machine, you want
+to mount the Docker socket from inside the VM, and not mount `/var/run/docker.sock` from the host filesystem (using
```

Comment:
If we mention this paragraph, I think we should also mention the limitations of using the raw socket vs the docker socket.

I remember when I was developing Volumes Backup that I had issues when reading Windows paths because the raw socket will not transform them correctly.

So I actually ended up using `/var/run/docker.sock` in the compose file. /cc @gtardif

Reply:
I created a ticket in our backlog to improve this guide. I added a link to this comment so that we don't forget about it.